### ISSUE TITLE
Allow JSON:API to deliver image style information

### ISSUE THREAD

**Issue by timcosgrove** [department-of-veterans-affairs/va.gov-cms#8913]
## Description
We need to expose a way for Next to retrieve images with identified Drupal image styles. There are a few paths here. Notes on modules and approaches:
* https://www.drupal.org/project/jsonapi_image_styles
* https://www.drupal.org/project/consumer_image_styles
* https://www.drupal.org/project/drupal/issues/2825812
* https://www.lullabot.com/articles/decoupled-drupal-hard-problems-image-styles

## Acceptance Criteria
- [x] Above approaches to incorporating image styles into JSON:API are reviewed and a decision made on approach and documented
  - consumer_image_styles chosen: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8913#issuecomment-1121779859 
- [x] JSON:API delivers the image styles using the selected approach
  - https://prod.cms.va.gov/jsonapi/media/image?include=image&page[limit]=5 ; scroll down to `included` section of response to see image style information added to each image file.

### CMS Team
Please check the team(s) that will do this work.

- [X] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by timcosgrove** [comment]
Different approaches were reviewed.

* [ImageItem should have an "derivatives" computed property, to expose all image style URLs](https://www.drupal.org/project/drupal/issues/2825812) - This is not in a stable state and has not been worked on for some time. If this is merged into core at a later time, we can reconsider taking advantage of this.
* [consumer_image_styles](https://www.drupal.org/project/consumer_image_styles) - good approach, but overly cumbersome for what we're currently trying to achieve. If we progress to a Public Content API with several known consumers/clients, we might consider using this.
* [jsonapi_image_styles](https://www.drupal.org/project/jsonapi_image_styles) - Lightweight, zero config, low impact module. It does only what we need and no more.

Once merged, image style URIs will become available in the JSON:API response for image files. This can be viewed for example at `/jsonapi/media/image?include=image&page[limit]=2` on a server where the module is enabled (scroll down to `included`, at bottom). 

In Next-drupal terms, one would need to include not just the Media entity but also the file that is attached to it, in order for these URIs to become available. So for example, requests for the entire collection of`node--story` data would need the following includes:
```
  const storyNodes = await drupalClient.getResourceCollectionFromContext<
    DrupalNode[]
  >('node--story', context, {
    params: {
      include:
        'field_media, field_media.thumbnail, field_media.image',
      sort: '-created',
      'filter[status][value]': '1',
      page: {
        limit: 5,
      },
    },
  })
```
This would make the image URI in `crop_2_1` style for an individual node available at
```
node.field_media.image.image_style_uri.crop_2_1
```

**Comment by timcosgrove** [comment]
Viewable on the PR server: https://cms-glsd8ns5dfzcqg4k8njo6yxfbco0j3cj.ci.cms.va.gov/jsonapi/media/image?include=image&page[limit]=2 (scroll down to `included`)

**Comment by timcosgrove** [comment]
Update is merged; waiting on merge to do a bit more setup on prod, since Consumers are database-stored entities.
